io.ossia.score.yml

```diff
@@ -39,8 +39,10 @@ finish-args:
   - --filesystem=xdg-run/pipewire-0
 
   # Plugins
-  - --env=VST_PATH=/app/extensions/Plugins/vst
-  - --env=VST3_PATH=/app/extensions/Plugins/vst3
+  - --env=LV2_PATH=$HOME/.lv2:/app/extensions/Plugins/lv2:/app/lib/lv2
```

Comment:
this doesn't work. the `$HOME` will be literally in the variable content.

If you want to set these, set then in a wrapper script.

Reply:
First are the variables used? LV2_PATH is because that the standard LV2. But the other it's a crap shoot.

`LV2_PATH` must always be set in a wrapper script because it **require** `$HOME`. If not it is a bug.

For there other, a lot of time it needs to be patched. Anything using JUCE for example.

The VST3 SDK has even worse values as it assume that `/usr/local` always exist and doesn't allow anything else. There is an open issue but they seem uninterested in supporting Linux properly.

There is almost never any need to use `$HOME` for any of the other plugin format.

In short it's a case by case basis.

Reply:
> This second one will do nothing.... (there is no variable). 
my bad, I meant of course`export VST_PATH=$HOME[etc etc...]`

Reply:
> for $HOME how else is it supposed to work for all the plug-ins that install in ~/.vst ~/.vst3 ~/.lv2 ? it seems fairly standard as a place to look for those plug-ins, at least my users asked me more than once for this..

most implementation don't care of the variable and will always looks there. I don't know about your case. LV2 requires it in the path, but for other reason.

Reply:
> The VST3 SDK has even worse values as it assume that /usr/local always exist and doesn't allow anything else. There is an open issue but they seem uninterested in supporting Linux properly.


hmm I see - I patch a few things out of the VST3 SDK to support funkier cases such as on ARM64 and such, and do my own search anyways. I think it's becoming more prevalent as I've seen recent versions of both Bitwig and Ableton Live provide the ability to customize and override VST3 search folders...

Reply:
in the last version of the software I update to make it also look in ~/.{plugin_format}